mesonbuild/dependencies/base.py

```diff
@@ -37,6 +37,7 @@
 from ..mesonlib import MachineChoice, MesonException, OrderedSet, PerMachine
 from ..mesonlib import Popen_safe, version_compare_many, version_compare, listify
 from ..mesonlib import Version
+from typing import List, Tuple
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'List' from line 29

<sub>posted by Sider</sub>